src/sass/components/_plugins.scss

```
@@ -39,6 +54,239 @@
   }
 }
 
+#my-page.dashboard {
+  .mypage-box:not(.block-welcome, .block-projectinformation, .block-text, .block-text_async) {
+    margin-bottom: $line-height-computed;
+    padding: $panel-body-padding;
+    border: 1px solid $panel-border;
+    border-radius: $border-radius-base;
+    background-color: $panel-bg;
+    color: $panel-color;
+    font-size: .92em;
+    word-wrap: break-word;
+
+    > h3 {
+      margin-bottom: $panel-body-padding-vertical;
+    }
+  }
+}
+
+.controller-rdb_taskboard .gototop {
+  margin-right: $padding-side;
+}
+
+//
+// Additional tags
+// https://www.redmine.org/plugins/additional_tags
+// --------------------------------------------------
+
+.issue .tags.attribute {
+  padding-left: 160px;
+
+  .label {
+    width: 160px;
+    margin-left: -160px;
+  }
+
+  #tags-data {
+    padding-top: 3px;
+  }
+}
+
+#main {
+  .additional-tag-label-color {
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+
+  #admin-menu a.additional-tags::before {
+    font-size: 1em;
+  }
+}
+
+#wiki_tags,
+#edit-tags-form {
+  .select2-container {
+    margin-bottom: 5px;
+
+    &.select2-container--default .select2-selection--multiple .select2-selection__rendered {
+      margin: 0;
+      padding: 3px;
+    }
+  }
+}
+
+
+//
+// Redmine Dashboard
+// https://www.redmine.org/plugins/redmine-dashboard
+// --------------------------------------------------
+
+#rdb-taskboard {
+  #rdb-header {
+    a#rdb-refresh {
+      padding: 2px 0;
+      border-width: 0;
+      background-image: none;
+      line-height: 1.2em;
+
+      &::before {
+        content: "\f021";
+        display: block;
+        width: 1em;
+        margin: 0 auto;
+        color: $green;
+        font-family: FontAwesome;
+        line-height: 1.2em;
+      }
+    }
+
+    .rdb-board .rdb-menu h2 {
+      font-size: 1em;
+    }
+
+    .rdb-board,
+    .rdb-filter,
+    .rdb-option {
+      margin-top: 4px;
+      margin-bottom: 0;
+    }
+  }
+
+  a#rdb-refresh,
+  a#rdb-refresh:focus,
+  a#rdb-refresh:hover,
+  .rdb-menu-link,
+  .rdb-menu-link:focus,
+  .rdb-menu-link:hover,
+  .rdb-menu .rdb-list a,
+  .rdb-menu .rdb-list a:focus,
+  .rdb-menu .rdb-list a:hover {
+    color: $gray-900;
+    text-decoration: none;
+  }
+
+  div:not(.rdb-card-header, .rdb-compact-header, .rdb-card-title) .rdb-menu {
+    .rdb-list {
+      > ul > li > a,
+      .rdb-multicheck a {
+        padding: 3px 5px;
+      }
+
+      .rdb-checkbox-link.rdb-checkbox-link-enabled {
+        background-image: none;
+
+        &::before {
+          content: "\f00c";
```

Use appropriate `$fa-var-`, they're descriptive

```suggestion
          content: $fa-var-check;
```

This applies to all places where icons were added.

---

Fixed!